transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java

```diff
@@ -237,7 +236,7 @@ private int epollWait(boolean oldWakeup) throws IOException {
         long totalDelay = delayNanos(System.nanoTime());
         int delaySeconds = (int) min(totalDelay / 1000000000L, Integer.MAX_VALUE);
         return Native.epollWait(epollFd, events, timerFd, delaySeconds,
-                (int) min(totalDelay - delaySeconds * 1000000000L, Integer.MAX_VALUE));
+                (int) min(MAX_SCHEDULED_TIMERFD_NS, min(totalDelay - delaySeconds * 1000000000L, Integer.MAX_VALUE)));
```

Comment:
I think this can be simplified to drop Integer.MAX_VALUE?

Reply:
Sorry not sure I understand... what exactly you propose ?

Reply:
```java
return Native.epollWait(epollFd, events, timerFd, delaySeconds,
    (int) min(MAX_SCHEDULED_TIMERFD_NS, totalDelay - delaySeconds * 1000000000L));
```

Reply:
ah yeah... let me do this

Reply:
@carl-mastrangelo PTAL again